website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-setting-up-bigquery-oauth.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Setting up BigQuery OAuth"
+---
+
+:::info Enterprise Feature
+
+This guide describes a feature of the dbt Cloud Enterprise plan. If you’re interested in learning more about an Enterprise plan, contact us at sales@getdbt.com.
+
+:::
+
+dbt Cloud supports [OAuth authentication](https://cloud.google.com/bigquery/docs/authentication) with BigQuery. When BigQuery OAuth is enabled, users can interact with the BigQuery warehouse as individual users, rather than leveraging a shared service level authentication. 
+
+:::info Some Pre-Work Required
+
+Before setting up a Client ID & Secret, you'll have to have your existing BigQuery Settings in order. We recommend using a Service Account JSON file, and have a walkthrough for that [here](https://docs.getdbt.com/tutorial/setting-up/#generate-bigquery-credentials)
+
+:::
+
+### Configuring a Client ID & Secret
+To enable BigQuery OAuth, you will need a Client ID & Secret for [authentication](https://cloud.google.com/bigquery/docs/authentication) with BigQuery to manage the OAuth connection between dbt Cloud and BigQuery.
+
+In the BigQuery console you'll want to navigate to the Credentials page:
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/bq_oauth/bq_oauth_creds_bq_sidebar.png" title="BigQuery Sidebar Menu to Credentials Page" />
+
+There you'll see your existing Keys, Client IDs and Service Accounts - you'll want to click the "Create Credentials" button at the top and follow the steps, like this:
+
+<Lightbox src="/img/docs/dbt-cloud/dbt-cloud-enterprise/gsuite/bq_oauth/create_creds_bq_oauth.gif" title="Creating OAuth Credentials in BigQuery" />
+
+For the fields we recommend the following:
+- **Application Type:** Web application
+- **Name:** dbt cloud
+- **Authorized JavaScript origins:** Not required, but some organizations may have URIs they'd like to use here
+- **Authorized redirect URIs:** Not required, but some organizations may have URIs they'd like to use here
```

Comment:
Same as above - this field is required. I think we want to advise users to use a redirect URI that looks like:
```
https://cloud.getdbt.com/complete/bigquery
````

If the user is running dbt Cloud somewhere other than cloud.getdbt.com (ie. single-tenant or on-prem), then they should replace `cloud.getdbt.com` with the hostname of their Cloud instance.